system/security/shared/caching.cpp

```diff
@@ -556,6 +556,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (!fullScope || !*fullScope)
     {
         *accessFlags = queryDefaultPermission(sec_user);
+        WARNLOG("FileScope unspecified for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
```

Comment:
Doesn't this mean root scope, rather than no scope specified?
Is it possible to define the permissions for the root scope?

Reply:
I don't think there really is the concept of a root scope. If no scope is specified, then default scope is applied. That implies that there are no permissions for a root scope other than default.

Reply:
After some discussion with Russ, while there is not a formal concept of a root scope, this case is in essence an attempt to access a file at the root. There is probably a lot of code that takes advantage of this. What is your feeling on whether we should log this or not? I err on keeping the log statement since default permissions are being applied. That could be useful in researching a workunit failure.

Made a change to the log message, but left 'Filescope' as first word to maintain consistency in file scope log messages.